ui/v1/maslowWorker.js

```diff
@@ -4,15 +4,16 @@ import * as api from '@jsxcad/api-v1';
 import * as convertThree from '@jsxcad/convert-threejs';
 import * as sys from '@jsxcad/sys';
 
-import { assemble, difference, intersection, union } from '@jsxcad/api-v1-shape';
+import { assemble, intersection, union } from '@jsxcad/api-v1-shape';
 import { clearCache } from '@jsxcad/cache';
 import { hull } from '@jsxcad/api-v1-extrude';
-import { pack } from '@jsxcad/api-v1-layout';
+// import { pack } from '@jsxcad/api-v1-layout';
 import { toStl } from '@jsxcad/convert-stl';
 import { toSvg } from '@jsxcad/convert-svg';
 
```

Comment:
It occurs to me that you might write something like

const shape = geometry => api.Shape.fromGeometry(geometry); // Since this gets written out a lot.

const interface = {
  'bounding_box': g => shape(g).measureBoundingBox(),
  'circle': (diameter, sides) => Circle.ofDiameter(diameter, { sides }).toDisjointGeometry(),
  ... //
}

Then call

return (interface(key) || error)(...values);

would probably do the trick